docs/docs/how-gatsby-works-with-github-pages.md

```diff
@@ -2,7 +2,7 @@
 title: How Gatsby Works with GitHub Pages
 ---
 
-GitHub pages is a service offered by GitHub that allows hosting for websites configured straight from the repository. A Gatsby site can be hosted on GitHub pages with just a few configurations to the codebase and the repository's settings.
+GitHub pages is a service offered by GitHub that allows hosting for websites configured straight from the repository. A Gatsby site can be hosted on GitHub pages with having just a few tweaks to the codebase as well as the repository settings.
 
```

Comment:
@gillkyle  repository's settings sounds a bit awkward right :thinking:

Reply:
Well I'm probably biased because I wrote this 😅 maybe it would be good to have someone else weigh in.

Reply:
@sidharthachatterjee thoughts :thinking:

Reply:
cc @pieh @DSchau

Reply:
I think I agree with @gillkyle here that while these changes aren't wrong:

- they don't meaningfully _correct_ an incorrect statement 
- the improvement is a function of style so both are equally good 

In such a case, I prefer to defer to _not accepting_ a change in an effort to reduce churn. That's not to say that your work isn't valuable so thank you for taking the time to open this nonetheless!

Reply:
I'm not native english speaker, so I'll excuse myself from this discussion ;)